src/main.rs

```diff
@@ -206,13 +210,49 @@ fn expand_dir(path: &str) -> anyhow::Result<Vec<String>, io::Error> {
         .collect())
 }
 
-fn open_editor(input_files: &[String], editor_string: &str) -> anyhow::Result<Vec<String>> {
+/** Split path into directory path and filename. */
+fn path_and_file_name(line: &String) -> Option<(PathBuf, String)> {
+    let path = PathBuf::from(line);
+    let dirname = path.parent().and_then(|p| Some(PathBuf::from(p)));
+    let file_name = path
+        .file_name()
+        .and_then(|f| f.to_os_string().into_string().ok());
+
+    match (dirname, file_name) {
+        (Some(d), Some(f)) => Some((d, f)),
+        _ => None,
```

Comment:
If I give `local_path` (without a directory), that function seems to return None, which is then skipped in `filter_map`.. why?

IMO the features should support this too, no?

Reply:
Are you saying that running `renamer -n local-file.txt` isn't working for you?

Reply:
My mistake, I was only looking at the code, I tested it and it actually work although I don't really understand why...

When I run `renamer -n Cargo.toml src/main.rs` what is dirname for `Cargo.toml` ?

Reply:
It's an empty string